src/main/java/com/fabriccommunity/spookytime/mixin/MixinAbstractSkeletonEntity.java

```diff
@@ -0,0 +1,25 @@
+package com.fabriccommunity.spookytime.mixin;
+
+import com.fabriccommunity.spookytime.common.SpookyItems;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.mob.AbstractSkeletonEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.LocalDifficulty;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Random;
+
+@Mixin(AbstractSkeletonEntity.class)
+public class MixinAbstractSkeletonEntity {
+    @Inject(method = "initEquipment", at = @At(value = "TAIL"))
+    protected void initEquipment(LocalDifficulty ld, CallbackInfo cb) {
+        Random random = new Random();
+        if (random.nextInt(3) == 0) {
```

Comment:
Should probably hike up the rarity to ~1/30 or so. Not sure if there's dimension context in the entity at this stage or not, but if there is, you could have a higher chance only in the spooky dimension (and lower everywhere else).